src/main/kotlin/nl/info/client/kvk/model/KvkSearchParameters.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos, 2025 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package nl.info.client.kvk.model
+
+import jakarta.ws.rs.QueryParam
+import java.math.BigDecimal
+
+// Would like to use a data class here but unfortunately cannot get that to work with the JAX-RS @BeanParam annotation
```

Comment:
I tried converting this to a Kotlin data class but I could not get that to work with JAX-RS's @BeanParam.. I asked the AI and it replied something like: "that will probably never work, better to stick with a plain class with vars.."